README.md

```diff
@@ -63,20 +63,22 @@ cryptic, but may look like: `missing attribute name in .leapp/info`, etc.)
 ```bash
 $ make register
 ```
+
 where `register` target will run `snactor repo find --path repos`
 (you can verify if your repositories are registered in
 `~/.config/leapp/repos.json`).
 
 ## Troubleshooting
 
 ### Where can I report an issue or RFE related to the framework or other actors?
+
 - GitHub issues are preferred:
   - Leapp framework: [https://github.com/oamg/leapp/issues/new/choose](https://github.com/oamg/leapp/issues/new/choose)
   - Leapp actors: [https://github.com/oamg/leapp-repository/issues/new/choose](https://github.com/oamg/leapp-repository/issues/new/choose)
 
 - When filing an issue, include:
   - Steps to reproduce the issue
-  - The logs `/tmp/leapp-report.txt`, `/tmp/download-debugdata` and `/var/log/upgrade.log`
+  - All the logs in `/var/log/leapp/`
```

Comment:
Actually as I'm reading it I think somebody could skip including the dnf-debugdata as it is technically not a log.

Reply:
Maybe synchronize it with whatever you have in bug template.